module-2/backend/week_one.md

```diff
@@ -5,30 +5,102 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+* GET - client requests information
+
+* PUT - client sends information
+
+* POST - works in conjunction with PUT
+
+* PATCH - updates information
+
+* DELETE - destroys information
+
 2. What is Sinatra?
+
+A lightweight web framework written in Ruby based on Rack
+
 4. What is MVC?
+
+MVC stands for Model, View, Controller, a systematic approach to object oriented web design
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+In order to prepare for all the things Rails offers us, and that web conventions are normally the same across all frameworks
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+Params
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+
+@name = "Mr. Ed"

```

Comment:
The `@` makes this an instance variable. To access a local variable you need to pass `locals` as part of the options hash.

```ruby
get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```